x-pack/plugins/cloud_integrations/cloud_data_migration/server/plugin.ts

```diff
@@ -17,7 +17,6 @@ export class CloudDataMigrationPlugin implements Plugin<void, void, any, any> {
       management: {
         data: [PLUGIN_ID],
       },
-      catalogue: [PLUGIN_ID],
```

Comment:
__question:__ I assume this code wasn't released yet, right? Just checking if we're making a breaking change or not.

Reply:
No it wasn't

Reply:
Correct, it wasn't released